R/modify-model-field.R

```diff
@@ -177,7 +183,23 @@ remove_tags <- function(.mod, .tags) {
     .value = .tags,
     .append = FALSE,
     .remove = TRUE,
-    .char_value = TRUE,
+    .char_value = TRUE
+  )
+}
+
+#' @title  set_star_status
+#' @param .star boolean variable to indicate significance
+#' @param .mod list object of model data
+#' @describeIn Add or remove star on a model object and corresponding YAML.
+#' @export
+set_star_status <- function(.mod, .star) {
```

Comment:
I see the appeal of having a single function, but given the other `modify_model_field` functions, but I'd vote for `remove_star(.mod)` and `add_star(.mod)`, as I mentioned in a [previous comment](https://github.com/metrumresearchgroup/bbr/pull/487#discussion_r883629404).

Perhaps check with @seth127 to see what his thoughts are on this.